example-article.json

```diff
@@ -76,6 +104,35 @@ This file specifies what posts look like:
     ]
 }
 
+
+# Example 
+# http://service/posts/{post_id}/comments access to the comments in a post
+# http://service/posts/{post_id}/comments?page=4
+# http://service/posts/{post_id}/comments?page=4&size=40
+{
+	"query": "comments",
```

Comment:
Again, shouldn't need a query here. The url + method should be enough to specify what is happening.